src/Command/InitCommand.php

```diff
@@ -100,7 +109,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if (isset($tempHostFile)) {
             $host = file_get_contents($tempHostFile);
         }
-        $hosts = explode(',', $io->ask('Hosts (comma separated)', $host));
+        $hosts = [];
+        $answer = $io->ask('Hosts (comma separated)');
```

Comment:
Why this change? Also looks like `$host` default value is missing.

Reply:
Behavior before this change:

```
 Hosts (comma separated):
 > 

 TypeError  in InitCommand.php on line 103:

  explode() expects parameter 2 to be string, null given
  ```
  
  Behavior after this change:
  
  ```
   Hosts (comma separated):
 > 

No hosts given, you need to define them manually in deploy.php.
Successfully created deploy.php
```

`$host` is initialized in line 108.

Reply:
$host is initialized but not used. It should go to `$io->ask('Hosts (comma separated)', $host)`.

Reply:
Ah i see. It is the default value of the potential guess. I added it to the appropriate place.